### DISCUSSION TITLE
New Action for Tasks in Schedules: Remove Files

### DISCUSSION THREAD

**Discussion by Maelep** [pterodactyl/panel#3697] [Feature Requests]
Add the `Deletes Files` option in the Action drop down menu when creating a new task that will allow users to delete a list of files.
This action displays a text box allowing you to list paths to files on the server that you wish to delete. For example:
```
./server/rust/*.map
./server/rust/*.sav
./server/rust/*.sav.1
./server/rust/*.sav.2
./server/rust/player.blueprints.*.db
./server/rust/player.blueprints.*.db-journal
```

**Comment by trenutoo** [comment]
[There's an open feature request for this](https://github.com/pterodactyl/panel/issues/2422)

**Reply by Maelep** [reply]
It isn't the same. I'm suggesting selected file suppression as Task. Removing all files is already possible with the installation script.

**Reply by trenutoo** [reply]
> It isn't the same. I'm suggesting selected file suppression as Task. Removing all files is already possible with the installation script.

My bad. I linked the wrong issue. Open a feature request on the issues instead, that way I and others can track it. This is something I personally plan to implement after V2.

**Reply by Maelep** [reply]
I don't think to understand. You would like that I suggest my features on the issue you linked?

**Reply by trenutoo** [reply]
> I don't think to understand. You would like that I suggest my features on the issue you linked?

Open a feature request on the [Github repository issues](https://github.com/pterodactyl/panel/issues). It's much easier to keep track of that way.

**Reply by Maelep** [reply]
Done: https://github.com/pterodactyl/panel/issues/3702#issue-1033862692
Thank you for your answer.
